web/sharings/sharings.go

```diff
@@ -186,6 +187,28 @@ func RevokeSharing(c echo.Context) error {
 	return c.NoContent(http.StatusNoContent)
 }
 
+// RevokeRecipient is used by the owner to revoke a recipient
+func RevokeRecipient(c echo.Context) error {
+	inst := middlewares.GetInstance(c)
+	sharingID := c.Param("sharing-id")
+	s, err := sharing.FindSharing(inst, sharingID)
+	if err != nil {
+		return wrapErrors(err)
+	}
+	_, err = checkCreatePermissions(c, s)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusForbidden)
+	}
+	index, err := strconv.Atoi(c.Param("index"))
+	if err != nil || index == 0 || index >= len(s.Members) {
```

Comment:
If the index starts at 1, it might be useful to say it in the documentation

Reply:
`index == 0` is the sharer him-self

Reply:
Yes, but it might be not obvious for a dev not familiar with the sharing, that would have to implement the revocation and would just read the doc of this route :)

Reply:
Yep -> https://github.com/cozy/cozy-stack/pull/1384/commits/9436ba11b37b823d996b03406146c50432cf47fa